src/classes/CON_Email.cls

```diff
@@ -281,11 +281,24 @@ public class CON_Email {
                 'Work Email' => 'Work',
                 'Alternate Email' => 'Alternate'
             };
-
+                     
+            String namespace = UTIL_Namespace.getNamespace();
+            String prefix = (String.isNotBlank(namespace)) ? namespace + '__' : '';
```

Comment:
We also support custom fields (user would add a value to the picklist and the field to the layout). Would that functionality be affected?

Reply:
That functionality would work if the label and picklist have an exact match.

Reply:
So the prefix won't be added for fields that are not in the package? 

Ideally we would test this still works, but I guess the only way to test it is after merging (with a beta). Unfortunately namespaced scratch orgs don't help here...

Reply:
Hmm we are appending the prefix for only "PreferredPhone__c" and "Preferred_Email__c" fields which are packaged but we are not appending it to the actual phone/email fields. Please let me know if I that was your question.

Reply:
Ah, sorry. My bad. I looked at this too quickly. Yes, we are applying the prefix only to the field, not to the picklist values (which I guess don't need it anyway). Thanks for bearing with me, Samsheeth!